src/fix_hmc.cpp

```diff
@@ -684,17 +684,17 @@ double FixHMC::compute_vector(int item)
     return DeltaKE;
   else if (n == 4)
     return DeltaPE + DeltaKE;
-  else if (n == 5) {
-    double *dx, local_msd = 0.0;
-    int nlocal = atom->nlocal;
-    for (int i = 0; i < nlocal; i++) {
-      dx = deltax[i];
-      local_msd += dx[0]*dx[0] + dx[1]*dx[1] + dx[2]*dx[2];
-    }
-    double msd;
-    MPI_Allreduce(&local_msd,&msd,1,MPI_DOUBLE,MPI_SUM,world);
-    msd /= atom->natoms;
-    return msd;
+  // else if (n == 5) {
```

Comment:
Does deleting commented-out code still speed it up? Either way, I agree this should be taken out. Good find.